webpack.config.js

```diff
@@ -7,40 +7,35 @@ var UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 
 var isDev = JSON.stringify(process.env.NODE_ENV !== 'production');
 
-var banner = (
+var banner =
   '/**\n' +
-  ' * FixedDataTable v' + packageJSON.version + ' \n' +
+  ' * FixedDataTable v' +
```

Comment:
supernit: although the output will be the same, I prefer the earlier format. I wonder if there're more such cases in other files.

Maybe we should have prettier ignore this section, or define this using template literals.

Reply:
Ignored